### DISCUSSION TITLE
Marketplace: README from main branch - `Use latest version` points to a tag

### DISCUSSION THREAD

**Discussion by jbergstroem** [community/community#38725] [Actions]
### Select Topic Area

Product Feedback

### Body

The readme for a marketplace action ([example - my own](https://github.com/marketplace/actions/hadolint-github-action)) sources from the default branch but all interactions with the action (read: start using it) references the latest tag. This can be misleading. Consider:

### Header info - mentions tag
<img width="459" alt="Screenshot 2022-11-09 at 6 55 17 PM" src="https://user-images.githubusercontent.com/176984/200950432-602e067e-f3d6-457c-a222-669865355fdb.png">

### CTA button - points to tag
<img width="502" alt="Screenshot 2022-11-09 at 6 55 12 PM" src="https://user-images.githubusercontent.com/176984/200950491-605fcd37-da1a-4366-b5b8-db3bab3b16bb.png">

### Readme content

..uses [latest version from main](https://github.com/jbergstroem/hadolint-gh-action/blob/main/README.md), worth comparing to the one that the user [would be actually using](https://github.com/jbergstroem/hadolint-gh-action/blob/v1.10.0/README.md) should they use the latest tag.

Here's the visual diff between these two versions (`main`, unreleased and `v1.10.0`, what the user is pointed to):

<img width="996" alt="Screenshot 2022-11-09 at 7 01 47 PM" src="https://user-images.githubusercontent.com/176984/200951449-609ee2eb-1c07-418a-93d9-c15d1d42cd99.png">

I find this concerning seeing how what the user would expect is different from the truth. In this particular case, the action would fail to run.

### Suggested solution

Refer (and possibly link) to the readme from said tag if that is what you're telling the user to use.

**Comment by ossanna16** [comment]
Hi there @jbergstroem and welcome to our community! Thank you for the feedback!
